src/coreclr/scripts/genLttngProvider.py

```diff
@@ -88,6 +88,34 @@
         "win:Binary"        :"const BYTE"
         }
 
+monoLttngDataTypeMapping ={
+        #constructed types
+        "win:null"          :" ",
+        "win:Int64"         :"const int64_t",
+        "win:ULong"         :"const uint32_t",
+        "win:count"         :"*",
+        "win:Struct"        :"const uint8_t *",
+        #actual spec
+        "win:GUID"          :"const int32_t",
+        "win:AnsiString"    :"const uint8_t*",
+        "win:UnicodeString" :"const uint8_t*",
+        "win:Double"        :"const double",
+        "win:Int32"         :"const int32_t",
+        "win:Boolean"       :"const bool",
+        "win:UInt64"        :"const uint64_t",
+        "win:UInt32"        :"const uint32_t",
+        "win:UInt16"        :"const uint16_t",
+        "win:UInt8"         :"const uint8_t",
+        "win:Pointer"       :"const size_t",
+        "win:Binary"        :"const BYTE"
```

Comment:
A couple of things you need to map differently:

"win:AnsiString"    :"const char*",
"win:UnicodeString" :"const ep_char8_t*"

"win:Pointer"       :"const void*",
"win:Binary"        :"const uint8_t"

Reply:
Thanks, done!